test/utils/index.ts

```diff
@@ -20,6 +30,23 @@ export function sanitizeFontSource(data: FontFaceData[]) {
   }))
 }
 
+type RemoteProviders = keyof typeof import('../../src/providers')
+type ProvidersWithOptimizer = Extract<RemoteProviders, 'google' | 'googleicons'>
+
+export function getOptimizerIdentityFromUrl(provider: ProvidersWithOptimizer, url: string) {
+  if (provider === 'google' || provider === 'googleicons') {
+    const params = new URL(url).searchParams
+
+    // Google Fonts provides 3 params: kit, skey, v.
+    // V changes frequently, but other 2 params are stable and suitable for identification.
+    return {
+      kit: params.get('kit') ?? '',
+      skey: params.get('skey') ?? '',
+    }
+  }
```

Comment:
In fact, only the v parameter is changed in [this CI](https://github.com/unjs/unifont/actions/runs/14361020445/job/40287557200).

Also, when I ran it earlier in my environment, v was further increased to 237.